x-pack/test/api_integration/apis/transform/index.ts

```diff
@@ -27,6 +27,7 @@ export default function ({ getService, loadTestFile }: FtrProviderContext) {
       await transform.testResources.resetKibanaTimeZone();
     });
 
-    loadTestFile(require.resolve('./delete_transforms'));
+    // loadTestFile(require.resolve('./delete_transforms'));
```

Comment:
nit: commented code

Reply:
Thanks for spotting this! Fixed in bcce968.